gui/default/assets/css/tree.css

```diff
@@ -56,3 +56,91 @@
     white-space: normal;
     word-break: break-all;
 }
+
+.fancytree-temp .fancytree-title {
+    opacity: 0.6;
+    font-style: italic;
```

Comment:
Italic is problematic for some languages (see https://github.com/syncthing/syncthing/blob/5f5d672a7d16c4e38a84e8af2feb7c04e1287212/gui/default/assets/css/overrides.css#L545-L550).

Maybe add

```
html[lang|="zh"] .folderTree-new-folder,
html[lang="ja"] .folderTree-new-folder,
html[lang|="ko"] .folderTree-new-folder {
    font-style: normal;
}
```

? The other way is to use `:not`, but the override seems simpler…